packages/astro/package.json

```diff
@@ -224,8 +224,8 @@
     "unified": "^11.0.4"
   },
   "engines": {
-    "node": ">=18.14.1",
-    "npm": ">=6.14.0"
+    "node": ">=18.20.1",
+    "npm": ">=10.5.0"
```

Comment:
Only 18% of our users are compatible with this change 😅

Reply:
I'll have to understand the logic of that... if Node.js whips `18` ships npm `10.5.0`, how can users use a lower version of npm? 😮

Reply:
On certain systems they don't ship together, you download (and upgrade) npm separately. In addition to that, it's the *latest* of Node 18 that ships with 10.5.0, the vast vast vast majority of people are not on the latest of 18, they're a version or two behind (reminder: last version of Node 18 came out 2 days ago)

Reply:
Thank you. I reverted the change